src/structures/Guild.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 
+const { deprecate } = require('util');
 const Base = require('./Base');
 const GuildAuditLogs = require('./GuildAuditLogs');
 const GuildPreview = require('./GuildPreview');
 const Integration = require('./Integration');
 const Invite = require('./Invite');
 const VoiceRegion = require('./VoiceRegion');
 const Webhook = require('./Webhook');
+const { Error } = require('../errors');
```

Comment:
I know nitpick, but can you re-add these back? Someone removed them at some point
```suggestion
const { Error, TypeError } = require('../errors');
```